examples/observers/litellm_example.py

```diff
@@ -1,14 +1,16 @@
 import os
 
 from litellm import completion
-from observers.observers import wrap_litellm
+
+from observers import wrap_litellm
+
 
 # Ensure you have both API keys set in environment variables
 os.environ["OPENAI_API_KEY"] = "your-api-key"
 os.environ["ANTHROPIC_API_KEY"] = "your-api-key"
 
 # Wrap the completion function to enable tracking
-completion = wrap_litellm(completion)
+client = wrap_litellm(completion)
```

Comment:
I believe this is just a callable and not a fancy client class: https://docs.litellm.ai/docs/

```suggestion
completion = wrap_litellm(completion)
```